wagtailio/settings/base.py

```diff
@@ -415,3 +418,5 @@
             "ZONEID": env["FRONTEND_CACHE_CLOUDFLARE_ZONEID"],
         }
     }
+
+WAGTAILCONTENTIMPORT_DOCX_PARSER = "wagtailio.utils.parsers.DocxHTMLParser"
```

Comment:
I can't actually see this file in the code - is it possible you accidentally didn't commit it?

Reply:
Thanks. I've added it now.